src/runtime/vm/executable.cc

```diff
@@ -86,7 +86,7 @@ PackedFunc Executable::GetFunction(const std::string& name, const ObjectPtr<Obje
   } else if (name == "vm_load_executable") {
     return PackedFunc([sptr_to_self, this](TVMArgs args, TVMRetValue* rv) {
       auto vm = make_object<VirtualMachine>();
-      vm->LoadExecutable(this);
+      vm->LoadExecutable(ObjectRef(sptr_to_self));
```

Comment:
GetObjectPtr<Executable>(this);

Reply:
As I know sptr_to_self is already ObjectPtr<Object>, if it is immediatly ObjectPtr<Executable> or can be simply converted to ObjectPtr<Executable> it will be nice, but it is not so. Unfortunately your approach is bad due to GetObjectPtr(...) creates new ObjectPtr<> object which contains copy of raw pointer. There is no any connection between origin and new ObjectPtr. It means that if origin object was released (e.g. when python session where it was generated is closed) we will have pointer to broken object and can not use it. just my PR solves this problem, but your approach does not.

Reply:
Sorry for the delayed reply. sptr_to_self actually is guanranteed to point to `this` in this particular case. So actually `GetObjectPtr<Executable>(...)` should work in this case. This is mainly because ObjectPtr is an intrusive pointer that directly leverages the object data structure. 

To confirm, we can add `ICHECK(sptr_to_self.get() == this);`